src/include/daos_srv/daos_engine.h

```diff
@@ -51,6 +51,13 @@ extern unsigned int	 dss_instance_idx;
 /** Bypass for the nvme health check */
 extern bool		 dss_nvme_bypass_health_check;
 
+/** Max number of vos targets per engine */
+#define			 DSS_MAX_VOS_TGT_CNT	96
+/* System xstream target ID */
+#define			 DSS_SYS_TGT_ID		1024
+/* Invalid target id */
+#define			 DSS_INVALID_TGT_ID	-1
```

Comment:
Don't define these in daos engine, define them in BIO or SMD.

Reply:
ok!